commitlint.config.ts

```diff
@@ -303,7 +303,8 @@ module.exports = {
                         let numRecomendations = 0;
                         let lowerCaseHeaderStr = headerStr.toLowerCase()
                         Object.entries(abbr).forEach(([key, value]) => {  
-                            if (lowerCaseHeaderStr.includes(key.toString())){
+                            let pattern = new RegExp("\\b(" + key.toString() + ")\\b")
```

Comment:
@realmarv are we sure this doesn't break other scenarios? to make sure, let's add these 2 tests in a 3rd commit:

a) ```title: 1 second``` should offer `1sec` suggestion
b) ```Configuration simplification``` should offer `config`

Reply:
Done